hasura/migrations/default/1681182034201_alter_table_public_contributions_drop_column_datetime_created/down.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."contributions" add column "datetime_created" timestamptz;
```

Comment:
This can be switched to a single alter table line, rathe than three:

```
ALTER TABLE "public"."contributions"
ADD COLUMN "datetime_created" timestamptz NOT NULL DEFAULT current_timestamp;
```